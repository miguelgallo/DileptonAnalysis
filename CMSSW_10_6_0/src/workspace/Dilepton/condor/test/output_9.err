Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
ImportError: No module named site
WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
Error in <TTree::SetBranchAddress>: unknown branch -> nGenMuonCand
Error in <TTree::SetBranchAddress>: unknown branch -> nGenMuonCandOutOfAccept
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_e
Error in <TTree::SetBranchAddress>: unknown branch -> nGenProtCand
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_e
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_status
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_mass
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_dpt
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_dphi
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_3Dangle

 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================
#0  0x00002b0b4657241c in waitpid () from /lib64/libc.so.6
#1  0x00002b0b464eff12 in do_system () from /lib64/libc.so.6
#2  0x00002b0b45e4852d in TUnixSystem::StackTrace() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#3  0x00002b0b45e4add4 in TUnixSystem::DispatchSignals(ESignals) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#4  <signal handler called>
#5  0x00002b0b53060e5c in ?? ()
#6  0x3fa5f5c280000000 in ?? ()
#7  0x3f914a3d80000000 in ?? ()
#8  0xbfd10f5c20000000 in ?? ()
#9  0xbfa8b33340000000 in ?? ()
#10 0x3fb92e1480000000 in ?? ()
#11 0xbfc13ae140000000 in ?? ()
#12 0x3f6d5c2900000000 in ?? ()
#13 0xbfbf47ae20000000 in ?? ()
#14 0x3fbed70a40000000 in ?? ()
#15 0x3fc723d700000000 in ?? ()
#16 0x3fa7000000000000 in ?? ()
#17 0xbf93e66660000000 in ?? ()
#18 0x3f86f0a3e0000000 in ?? ()
#19 0x3fd07851e0000000 in ?? ()
#20 0x0000000003238a20 in ?? ()
#21 0x00002b0b45acf000 in ?? ()
#22 0x0000000000000000 in ?? ()
===========================================================


The lines below might hint at the cause of the crash.
You may get help by asking at the ROOT forum http://root.cern.ch/forum
Only if you are really convinced it is a bug in ROOT then please submit a
report at http://root.cern.ch/bugs Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#5  0x00002b0b53060e5c in ?? ()
#6  0x3fa5f5c280000000 in ?? ()
#7  0x3f914a3d80000000 in ?? ()
#8  0xbfd10f5c20000000 in ?? ()
#9  0xbfa8b33340000000 in ?? ()
#10 0x3fb92e1480000000 in ?? ()
#11 0xbfc13ae140000000 in ?? ()
#12 0x3f6d5c2900000000 in ?? ()
#13 0xbfbf47ae20000000 in ?? ()
#14 0x3fbed70a40000000 in ?? ()
#15 0x3fc723d700000000 in ?? ()
#16 0x3fa7000000000000 in ?? ()
#17 0xbf93e66660000000 in ?? ()
#18 0x3f86f0a3e0000000 in ?? ()
#19 0x3fd07851e0000000 in ?? ()
#20 0x0000000003238a20 in ?? ()
#21 0x00002b0b45acf000 in ?? ()
#22 0x0000000000000000 in ?? ()
===========================================================


