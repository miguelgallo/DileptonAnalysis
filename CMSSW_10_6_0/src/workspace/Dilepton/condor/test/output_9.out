copying file
running
0cortes: 2887280
1cortes: 40613
2cortes: 37606
3cortes: 36258
4cortes: 16861
5cortes: 16861
6cortes: 10456
7cortes: 1400
8cortes: 19
moving file
